frontend/src/metabase/dashboard/analytics.ts

```diff
@@ -12,10 +12,25 @@ export const trackAutoApplyFiltersDisabled = (dashboardId: DashboardId) => {
   });
 };
 
-export const trackExportDashboardToPDF = (dashboardId: DashboardId) => {
+type DashboardAccessedVia =
+  | "internal"
+  | "public-link"
+  | "static-embed"
+  | "interactive-iframe-embed"
+  | "sdk-embed";
+
+export const trackExportDashboardToPDF = ({
+  dashboardId,
+  dashboardAccessedVia,
+}: {
+  dashboardId?: DashboardId;
+  dashboardAccessedVia: DashboardAccessedVia;
+}) => {
   trackSchemaEvent("dashboard", DASHBOARD_SCHEMA_VERSION, {
     event: "dashboard_pdf_exported",
-    dashboard_id: dashboardId,
+    // we send 0 because the snowplow table requires a value
```

Comment:
Can we make the field nullable instead?

Reply:
That was the initial plan but [from this thread](https://metaboat.slack.com/archives/C010L1Z4F9S/p1721746106222259?thread_ts=1721745811.912789&cid=C010L1Z4F9S) it seems that that would require a new table and we're trying to avoid that

Reply:
@alxnddr can merge it as is?